src/main/java/com/ryanharter/auto/value/moshi/AutoValueMoshiAdapterFactoryProcessor.java

```diff
@@ -126,6 +124,9 @@ private TypeSpec createJsonAdapterFactory(
         .addParameters(ImmutableSet.of(type, annotations, moshi))
         .returns(result);
 
+    // Avoid providing an adapter for an annotated type.
+    create.addCode("if (!$N.isEmpty()) return null;", annotations);
```

Comment:
I think you'll want `addStatement` here and omit the semi-colon. Otherwise you're missing a newline at the end.